packages/components/src/components/contract-card/contract-card-items/toggle-card-dialog.tsx

```diff
@@ -113,36 +105,28 @@ const ToggleCardDialog = ({
         />
     );
 
-    const toggle_wrapper =
-        should_show_cancellation_warning && isDesktop() ? (
-            <Popover
-                alignment='right'
-                classNameBubble='dc-contract-card-dialog__popover'
-                is_bubble_hover_enabled
-                margin={2}
-                zIndex='2'
-                message={
-                    <PopoverMessageCheckbox
-                        defaultChecked={should_hide_selected}
-                        checkboxLabel={getCardLabels().DONT_SHOW_THIS_AGAIN}
-                        message={getCardLabels().TAKE_PROFIT_LOSS_NOT_AVAILABLE}
-                        name='should_show_cancellation_warning'
-                        onChange={onPopoverCheckboxChange}
-                    />
-                }
-                onBubbleClose={onPopoverClose}
-            >
-                <div className='dc-contract-card-dialog-toggle__wrapper'>{edit_icon}</div>
-            </Popover>
-        ) : (
-            <div className='dc-contract-card-dialog-toggle__wrapper'>{edit_icon}</div>
-        );
-
     return (
         <div onClick={handleClick}>
-            <div ref={toggle_ref} className='dc-contract-card-dialog-toggle' onClick={toggleDialogWrapper}>
-                {is_valid_to_cancel ? toggle_wrapper : edit_icon}
-            </div>
+            {is_valid_to_cancel && should_show_cancellation_warning && isDesktop() ? (
+                <Popover
+                    alignment='right'
+                    classNameBubble='dc-contract-card-dialog__popover-bubble'
+                    className='dc-contract-card-dialog__popover'
+                    is_bubble_hover_enabled
+                    margin={2}
+                    zIndex='2'
+                    message={getCardLabels().TAKE_PROFIT_LOSS_NOT_AVAILABLE}
+                    onBubbleClose={onPopoverClose}
+                >
+                    <button ref={toggle_ref} className='dc-contract-card-dialog-toggle' onClick={toggleDialogWrapper}>
+                        {edit_icon}
+                    </button>
+                </Popover>
+            ) : (
+                <button ref={toggle_ref} className='dc-contract-card-dialog-toggle' onClick={toggleDialogWrapper}>
+                    {edit_icon}
```

Comment:
@farhan-nurzi-deriv previously, the condition was working a bit differently:
1. when `is_valid_to_cancel` was `false`, `edit_icon` was Not wrapped in `<div className='dc-contract-card-dialog-toggle__wrapper'>`;
2. when `is_valid_to_cancel` was `true` but `should_show_cancellation_warning` or `isDesktop()` were giving `false`, `edit_icon` was wrapped in `<div className='dc-contract-card-dialog-toggle__wrapper'>`.
Are we keeping this? Are you doing it be changing .scss?

Reply:
@maryia-deriv I removed that condition because i checked `dc-contract-card-dialog-toggle__wrapper` in the styles and we dont have any style for that. So i remove the wrapper from the edit icon